sa/storage-authority_test.go

```diff
@@ -240,91 +240,133 @@ func CreateDomainAuthWithRegID(t *testing.T, domainName string, sa *SQLStorageAu
 }
 
 // Ensure we get only valid authorization with correct RegID
-func TestGetLatestValidAuthorizationBasic(t *testing.T) {
-	sa, _, cleanUp := initSA(t)
+func TestGetValidAuthorizationsBasic(t *testing.T) {
+	sa, clk, cleanUp := initSA(t)
 	defer cleanUp()
 
-	// attempt to get unauthorized domain
-	authz, err := sa.GetLatestValidAuthorization(ctx, 0, core.AcmeIdentifier{Type: core.IdentifierDNS, Value: "example.org"})
-	test.AssertError(t, err, "Should not have found a valid auth for example.org")
+	// Attempt to get unauthorized domain.
+	authzMap, err := sa.GetValidAuthorizations(ctx, 0, []string{"example.org"}, clk.Now())
+	// Should get not results, but not error.
```

Comment:
There should probably be a `test.AssertNotError` here to check the above `err` then.